src/toolbarActions/toolbarList.js

```diff
@@ -239,7 +239,7 @@ const toolbarList = (state, dispatcher) => [
             { fn: () => downloadImg(s, d, 'PNG'), name: 'PNG' },
         ],
         visibility: true,
-        active: true,
+        active: state.curGraphInstance !== null,
```

Comment:
Isn't it enough to just say
active: state.curGraphInstance,
here?

(instead of stating !== null ?)

If that works, best to use the minimal form.

Reply:
yeah, I will look into it

Reply:
done @pradeeban